client-app/ui-kit/components/molecules/layout-with-right-sidebar/vc-layout-with-right-sidebar.vue

```diff
@@ -26,10 +26,14 @@ defineProps({
 
 <style lang="scss">
 .vc-layout-with-right-sidebar {
-  @apply flex flex-col;
+  @apply flex flex-col gap-5 px-6;
 
   @media (min-width: theme("screens.lg")) {
-    @apply flex-row items-start gap-x-6;
+    @apply flex-row items-start gap-x-6 px-0;
+  }
+
+  .vc-widget--size--lg {
+    @apply -mx-4.5 lg:mx-0;
   }
```

Comment:
Why is it here and how it's related to this layout?

Reply:
Good question, by design for a mobile view large widgets (sections) have different margins than small ones. Should we set these margins for each page separately? We need to think about this.